### PR TITLE
[Biography] Added organization

### PR DESCRIPTION
**Pull request by thomasbisignani**
This PR improves the old organization page and allows to manage the members via the BO.

### CODE REVIEW

File: app/config/services/admin.xml [hunk 1]
```diff
@@ -609,6 +609,19 @@
             </call>
         </service>
 
+        <!--
+        ######################################################
+        # Biography
+        ######################################################
+        -->
+        <service id="app.admin.biography_executive_office_member" class="AppBundle\Admin\ExecutiveOfficeMemberAdmin">
```
Comment (Remg) [inline app/config/services/admin.xml]: can we use the class as id ?

File: src/Controller/EnMarche/BiographyController.php [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+
+class BiographyController extends Controller
+{
+    /**
+     * @Route("/le-mouvement/notre-organisation", name="app_our_organization")
+     * @Method("GET")
+     */
+    public function executiveOfficeMemberListAction(ExecutiveOfficeMemberRepository $repository)
```
Comment (Remg) [inline src/Controller/EnMarche/BiographyController.php]: can you add a return type?

File: tests/Controller/EnMarche/PageControllerTest.php [hunk 3]
```diff
@@ -55,7 +55,6 @@ public function providePages()
         yield ['/emmanuel-macron/revolution'];
         yield ['/emmanuel-macron/videos'];
         yield ['/le-mouvement'];
-        yield ['/le-mouvement/notre-organisation'];
```
Comment (Remg) [inline tests/Controller/EnMarche/PageControllerTest.php]: why remove this line?

File: app/config/services/admin.xml (hunk 1 above)
Comment (thomasbisignani) [reply app/config/services/admin.xml]: Sonata needs this syntax for the security IMO.

**Pull request merged**